crd/overlayextensionconfig/manifests/acn.azure.com_overlayextensionconfigs.yaml

```diff
@@ -0,0 +1,66 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.13.0
+  name: overlayextensionconfigs.acn.azure.com
+spec:
+  group: acn.azure.com
+  names:
+    kind: OverlayExtensionConfig
+    listKind: OverlayExtensionConfigList
+    plural: overlayextensionconfigs
+    shortNames:
+    - oec
+    singular: overlayextensionconfig
+  scope: Namespaced
+  versions:
+  - additionalPrinterColumns:
+    - jsonPath: .spec.extensionIPRange
+      name: OverlayExtensionConfig IP range
+      priority: 1
+      type: string
+    name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: OverlayExtensionConfig is the Schema for the overlayextensionconfigs
+          API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: OverlayExtensionConfigSpec defines the desired state of OverlayExtensionConfig
+            properties:
+              extensionIPRange:
+                type: string
+            type: object
+          status:
+            description: OverlayExtensionConfigStatus defines the observed state of
+              OverlayExtensionConfig
+            properties:
+              message:
+                type: string
+              state:
+                enum:
+                - OECAccepted
```

Comment:
what is the difference between accepted and pending.

Reply:
Seems like there is no difference between accepted/pending in the design doc. Modified enum to have OECNone (default value), OECPending, OECSucceeded, OECFailed